_includes/components/indicator/units-template.html

```diff
@@ -2,7 +2,7 @@
   <% if(units.length) { %>
     <h4>{{ page.t.indicator.units_type }}</h4>
```

Comment:
@brockfanning I'm just looking at a feature branch for Phill and think this needs to be changed from `units_type` to `unit _of_measurement`